lib/proxy/http_download.rb

```diff
@@ -26,7 +26,7 @@ def initialize(src, dst, connect_timeout: DEFAULT_CONNECT_TIMEOUT, verify_server
       # keep last changed file attribute
       args << "--remote-time"
       # only download newer files
-      args += ["--time-cond", "file", dst.to_s]
+      args += ["--time-cond", dst.to_s]
```

Comment:
Looking at how it actually runs, perhaps this should be guarded with `if File.exist?(dst.to_s)` to avoid generating a warning.